### ISSUE TITLE
Add Accessibility (a11y) Test

### ISSUE THREAD

**Issue by jsmith-obrien** [jsmith-obrien/somersetrah-tests#8]
Given I include `cypress-axe` in my test setup
When I run `cy.injectAxe()` and `cy.checkA11y()` on the homepage
Then I should see any detected WCAG violations in the Cypress runner
